### ISSUE TITLE
Skybox texture doesn't load properly

### ISSUE THREAD

**Issue by TheCococQuartz** [wyskoj/midis2jam2#85]
**Describe the bug**
For some reason, some new textures added in 1.5 don't show up.
For example, the skybox, it's just the original black void it was before.
![AAAAAAA](https://user-images.githubusercontent.com/96637234/149844246-a66fb006-c45e-42ef-ac78-767b0be543e6.png)
This also includes every new drum texture (excluding power), which just displays the default blue texture.
The midi playing above ([this one](https://github.com/wyskoj/midis2jam2/files/7885007/THE-WORLD-REVOLVING.zip)) uses a drumkit other than the default and power one (idk the names, whichever one uses patch 25) and as you can see, still blue.

These textures are in the files, but for some reason don't show up.
Even weirder, when I edited the sky texture, the text I added to it showed up but the actual sky texture still doesn't.
![sans](https://user-images.githubusercontent.com/96637234/149847882-6780a267-28dc-499d-83b9-aef612cbd914.png)
![WHAT](https://user-images.githubusercontent.com/96637234/149847887-cdb9a6d0-3773-4d1c-a735-b673ccdeec03.png)
I tried this on the new drum textures but nothing happened

Maybe the retexturing function doesn't work or something, just a guess.

**System**
Windows 10

**Additional comment**
last bug report i swear

**Comment by wyskoj** [comment]
That drum kit (TR-808 Kit) doesn't have a special texture, it just uses the regular blue kit. However that skybox bug is very strange and I'm not sure what's causing that. What if you try making the skybox picture some other picture? Just to test and see if something like a normal photo or something appears correctly.

**Comment by TheCococQuartz** [comment]
I changed it to some forest picture and it was all black, no trace of anything.
![notnice](https://user-images.githubusercontent.com/96637234/149852201-a1a95e54-5ea5-4ab6-8504-4f31081c120c.png)

Not only that but I decided to take the edited texture I originally posted and paste it onto a different file. When I reopened that file the tiles where gone. It seems that the lighter tiles where replaced with the color of the darker tiles
![glowup](https://user-images.githubusercontent.com/96637234/149853385-37789cc2-5c00-4a56-8636-9348afd029cf.png)
(picture was downscaled to be readable btw)
